packages/projects/default-project/apartment.scene.json

```diff
@@ -997,12 +997,6 @@
         {
           "name": "visible",
           "props": {}
-        },
```

Comment:
the mount point needs to stay here. if this was due to saving the scene from the editor, then this is a bug that an be addressed elsewhere

Reply:
Yeah, not sure what happened there. Reverting.